src/index.ts

```diff
@@ -0,0 +1,97 @@
+import express from "express";
+import { User } from "./user";
+import { validateBody } from "./validation";
+
+const PORT = 3000;
+
+const app = express();
+
+app.use(express.json());
+
+const users: User[] = [];
+
+app.get("/users/:id", (req, res) => {
+  const user = users.find((user) => user.id === req.params.id);
+
+  if (!user) {
+    console.log(`User ${req.params.id} not found`);
+    return res.status(404).send({ message: "User not found" });
```

Comment:
this response is great but in your future projects please follow some standards like:
- use .json() instead of .send() for JSON data. (it is not bad or wrong if you want to know why I say this please read this https://medium.com/gist-for-js/use-of-res-json-vs-res-send-vs-res-end-in-express-b50688c0cddf)
- use structured response format like:
{
    "success": true,
    "data": [1, 2, 3]
}
- it is also good practice to use http status codes